project/Dependencies.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2018-2018 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+import sbt._
+
+object Dependencies {
+
+  object V {
+    val awsLambdaCore    = "1.2.0"
+    val awsLambdaEvents  = "2.2.2"
+    val kinesisClient    = "1.9.1"
+    val analyticsSdk     = "0.3.0"
+    val circe            = "0.9.3"
+    val hammock          = "0.8.5"
+    val specs2           = "4.2.0"
+    val scalacheckSchema = "0.1.0-M3"
```

Comment:
@chuwy do we know when 0.1.0 final will be out?

Reply:
I think nothing is blocking me - I can publish it today or whenever you want.